packages/arduino_ide.rb

```diff
@@ -19,47 +22,41 @@ class Arduino_ide < Package
   depends_on 'ant' => :build
   depends_on 'sommelier'
 
-  def self.build
     case ARCH
     when 'x86_64'
-      platform = 'linux64'
+      @platform = 'linux64'
     when 'i686'
-      platform = 'linux32'
+      @platform = 'linux32'
     when 'armv7l', 'aarch64'
-      platform = 'linuxarm'
+      @platform = 'linuxarm'
     end
```

Comment:
This section should be moved up above the description since they are instance variables.  Please use proper indentation.